### ISSUE TITLE
Firebase 9.1.0

### ISSUE THREAD

**Issue by cbraudo** [GameWorkstore/com.google.firebase.app#11]
Hi,

A new version of Firebase is available. Can you please update?

**Comment by snorrsi** [comment]
Hi @cbraudo ,

I’m currently on a vacation and therefore haven’t updated it yet. Will hopefully get to it tomorrow, latest on monday

**Comment by snorrsi** [comment]
Hi @cbraudo

Sorry that updating the packages took so long time. Finally updated Firebase SDK 9.1.0

Please let us know if you have any issue with the update.

**Comment by cbraudo** [comment]
Awesome, thanks!

**Comment by snorrsi** [comment]
Hi @cbraudo 

actually I noticed there was a build failure of the firebase.app package due to too big file.

Need to fix one package but the firebase.app size issue will take longer.

You and others should hopefully be able to use other packages. But you would need for now to include the firebase.app package locally using the tgz version.

**Comment by felipemullen** [comment]
Noticed today the plugins folder contains the incorrect files at this tag:

https://github.com/GameWorkstore/com.google.firebase.app/tree/9.1.0/Firebase/Plugins/x86_64

Files should have been the **9.1.0** versions, i.e. ` FirebaseCppApp-9_1_0.dll`, etc

**Comment by snorrsi** [comment]
@felipemullen we now of that issue.
Firebase added new .bundle file that was too big in size.
@HanzaRu had plans on checking it out this weekend, maybe cooking up a solution. Not sure of the state of that.

In the meantime you can you use the workaround solution I mentioned above (for firebase.app, others should work) with the official .tgz files from [https://developers.google.com/unity/archive](https://developers.google.com/unity/archive)

.. and curse Google, Firebase team for still haven’t bothered to create Git available packages like Google Play Plugins officially has

**Comment by HanzaRu** [comment]
Removed MacOS bundle, so we don't get stucked with this issue.

https://github.com/GameWorkstore/com.google.firebase.app/releases/tag/9.1.0

If Firebase team divided the library in 2, that would be solved on origin, but they don't listen to us :/

If someone needs it, just install the .bundle inside the project and it should be fine!

**Comment by HanzaRu** [comment]
@snorrsi mentioned that we can split the bundle in 2 bundles and that would solve the problem with macOS & arm64 builds,
so I gonna take a look, and if find a way to make it automatically, I will relaunch the version 9.1.0 with the fix this week.

--
cp FirebaseCppApp-9_1_0.bundle FirebaseCppApp-arm64-9_1_0.bundle
cp FirebaseCppApp-9_1_0.bundle FirebaseCppApp-x86_64-9_1_0.bundle
lipo -remove x86_64 FirebaseCppApp-arm64-9_1_0.bundle -o FirebaseCppApp-arm64-9_1_0.bundle
lipo -remove arm64 FirebaseCppApp-x86_64-9_1_0.bundle -o FirebaseCppApp-x86_64-9_1_0.bundle
--

**Comment by andnoonesthere** [comment]
@HanzaRu  Maybe it's better to publish 9.2.0 instead? :)

**Comment by HanzaRu** [comment]
> @HanzaRu Maybe it's better to publish 9.2.0 instead? :)

Yeah, i will try to schedule sometime in the weekend to work on this.

**Comment by snorrsi** [comment]
@HanzaRu any luck with it ? .. also has 9.3.0 been released by Firebase

**Comment by snorrsi** [comment]
@HanzaRu fyi I updated the UPM packages to 9.3.0 and am trying to figure out how we can upload the MacOS version

**Comment by snorrsi** [comment]
@HanzaRu I split the files using lipo as I mentioned. Created template for new meta file and copy that one over for the new name. Seems to work in my project at least.

Version 9.3.0 should work, let us know otherwise.
